apps/exceptions.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class UrlShortenerError(APIException):
```

Comment:
If URLShortenerClient is in common, why wouldn't this be too? Also, please keep naming/capitalization consistent where possible.

Reply:
I'll move this to python-common